kayttooikeus-service/src/main/java/fi/vm/sade/kayttooikeus/service/external/impl/OrganisaatioCache.java

```diff
@@ -50,6 +50,10 @@ public OrganisaatioPerustieto getRoot() {
         return getByOid(oid).map(org -> Stream.concat(org.parents(), org.andChildren())).orElse(Stream.empty());
     }
 
+    public Stream<OrganisaatioPerustieto> getAllOrganisaatios() {
+        return this.byOid.values().stream().flatMap(org -> Stream.concat(org.parents(), org.andChildren()));
```

Comment:
this.byOid.values() pitäisi riittää. Nyt taitaa tulla duplikaatteja mukaan.

Reply:
Joo korjataan tuo vielä ja sitten eteenpäin..